<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QWidget" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1651</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#account {
    background: transparent; /* tausta piirretään account.cpp paintEventissä */
    font-family: &quot;Segoe UI&quot;;
	font-size: 16px;
}

QLabel { 
	color: #F2F5FA;
    font-size: 28px;
    font-weight: 800;
    letter-spacing: 1px;
 }

#label_5, #label_6, #label_7, #label, #label_3, #labelSaldoCreditText, #labelSaldoLuottoText {
	font-size: 24px;
}
/* ATM-paneeli */
QFrame#atmCard {
    background-color: rgba(18, 20, 24, 220);
    border: 1px solid rgba(255, 255, 255, 30);
    border-radius: 18px;;
}

/* Header-alue */
QFrame#headerBar {
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #3A6DFF,
                                stop:1 #2449D8);
	border-radius: 18px;;
}

QLabel#bankTitle {
   color: #F2F5FA;
    font-size: 28px;
    font-weight: 800;
    letter-spacing: 1px;
}

QLabel#bankSubtitle, #label_2 {
    color: rgba(242, 245, 250, 170);
    font-size: 14px;
    font-weight: 700;
    letter-spacing: 2px;
}

/* Kortti/tili-info */
QLabel#labelLoginCardnumber {
    color: rgba(242, 245, 250, 170);
    font-size: 14px;
    font-weight: 700;
    letter-spacing: 2px;
}

/* Sisältöalue: stacked sivut */
QStackedWidget#stackedAccount {
    background: transparent;
    border: none;
}

/* Perus napit */
QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2A5CFF,
                                stop:1 #1B3FBF);
    color: #FFFFFF;

    border: 1px solid rgba(0, 0, 0, 60);
    border-radius: 12px;

    font-size: 16px;
    font-weight: 800;
    padding: 12px 14px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #3A6DFF,
                                stop:1 #2449D8);
}

QPushButton:pressed {
   background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #1E47D6,
                                stop:1 #16339E);
    border-bottom: 1px solid rgba(0, 0, 0, 110);
    padding-top: 13px;  /* pieni “painallus” */
    padding-bottom: 11px;
}

#btnLogOutVahvista, #btnNostaVahvistaVahvista, #btnNosta20, #btnNosta40, #btnNosta50, #btnNosta100, #btnNostaMuu, #btnTapahtumatVasen, #btnTapahtumatOikea {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #2ECC71,
                            stop:1 #1E9E57);
color: #FFFFFF;

border: 1px solid rgba(0, 0, 0, 60);
border-radius: 12px;

font-size: 16px;
font-weight: 800;
padding: 12px 14px;
}

#btnLogOutVahvista:hover, #btnNostaVahvistaVahvista:hover, #btnNosta20:hover, #btnNosta40:hover, #btnNosta50:hover, #btnNosta100:hover, #btnNostaMuu:hover, #btnTapahtumatVasen:hover, #btnTapahtumatOikea:hover {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #36D97A,
                            stop:1 #23B463);
}

#btnLogOutVahvista:pressed, #btnNostaVahvistaVahvista:pressed, #btnNosta20:pressed, #btnNosta40:pressed, #btnNosta50:pressed, #btnNosta100:pressed, #btnNostaMuu:pressed, #btnTapahtumatVasen:pressed, #btnTapahtumatOikea:pressed {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #1FAE60,
                            stop:1 #178548);
}

/* Footer-alue */
QFrame#footerBar {
    background-color: rgba(245, 247, 250, 235);
    border: 1px solid rgba(0, 0, 0, 35);
    border-radius: 18px;
}

/* Saldoboksit ja vahvistusboksi */
QLabel#labelSaldoSaldo,
QLabel#labelSaldoCreditLimit,
QLabel#labelNostaVahvistaSumma, #labelNostosumma {
    background-color: rgba(255, 255, 255, 245);
    color: #121418;

    border: 2px solid rgba(120, 140, 160, 180);
    border-radius: 10px;

    padding: 10px 12px;
    font-size: 18px;
}

QLabel#labelSaldoLuottoaJaljella {
	background-color: rgba(255, 255, 255, 245);
    color: #2ECC71;

    border: 2px solid rgba(120, 140, 160, 180);
    border-radius: 10px;

    padding: 10px 12px;
    font-size: 18px;
}


#labelNostosumma:focus {
	border: 2px solid rgba(40, 110, 220, 230);
    background-color: rgba(255, 255, 255, 255);
}

/* Virheilmoitukset */
QLabel#labelNostaValitseVirhe,
QLabel#labelNostaValitseKate,
QLabel#labelNostaValitseVirhe_2 {
    color: #FF3B30;                         /* kirkas punainen */
    background-color: rgba(255, 59, 48, 28); /* kevyt punainen tausta */
    border: 1px solid rgba(255, 59, 48, 90);
    border-radius: 10px;
    padding: 6px 10px;

    font-size: 13px;
    font-weight: 800;
}

/* Tapahtumataulun &quot;kortti&quot; */
QFrame#logsCard {
    background-color: rgba(255, 255, 255, 235);
    border: 1px solid rgba(0, 0, 0, 45);
    border-radius: 14px;
}

/* Tapahtumataulu */
QTableView#tableTapahtumat {
    background-color: rgba(255, 255, 255, 235);
    border: 1px solid rgba(0, 0, 0, 45);
    border-radius: 14px;
    gridline-color: rgba(0, 0, 0, 25);
    font-size: 14px;
	color: #121418;
}

/* Item-teksti varmistuksena (Qt joskus vaatii tämän) */
QTableView#tableTapahtumat::item {
    color: #121418;
    padding: 6px 8px;
}

/* Valinta ettei muutu “valkoiseksi valkoisella” tms. */
QTableView#tableTapahtumat::item:selected {
    background: rgba(58, 109, 255, 35);
    color: #121418;
}

QHeaderView::section {
    background-color: rgba(240, 242, 245, 245);
    color: #111111;
    font-weight: 900;
    border: none;
    padding: 8px;
}
</string>
  </property>
  <layout class="QGridLayout" name="outerGrid">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="centerLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>120</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QFrame" name="atmCard">
       <property name="minimumSize">
        <size>
         <width>520</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>580</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" name="atmLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>18</number>
        </property>
        <property name="topMargin">
         <number>18</number>
        </property>
        <property name="rightMargin">
         <number>18</number>
        </property>
        <property name="bottomMargin">
         <number>18</number>
        </property>
        <item>
         <widget class="QFrame" name="headerBar">
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <layout class="QHBoxLayout" name="headerLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>14</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>14</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="bankTextLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="bankTitle">
               <property name="text">
                <string>Kymppikerho Bank</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="bankSubtitle">
               <property name="text">
                <string>Itsepalveluautomaatti</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="headerSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelLoginCardnumber">
             <property name="text">
              <string>kortti • debit/credit</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedAccount">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>520</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>4</number>
          </property>
          <widget class="QWidget" name="screenLogin">
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>180</y>
              <width>420</width>
              <height>90</height>
             </rect>
            </property>
            <property name="text">
             <string>Tervetuloa Kymppikerho Bank:iin</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="screenSaldo">
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>210</y>
              <width>200</width>
              <height>40</height>
             </rect>
            </property>
            <property name="text">
             <string>Tilin saldo:</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelSaldoSaldo">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>210</y>
              <width>200</width>
              <height>40</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <fontweight>ExtraBold</fontweight>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>0 €</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="labelSaldoCreditText">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>280</y>
              <width>200</width>
              <height>40</height>
             </rect>
            </property>
            <property name="text">
             <string>Credit limit:</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelSaldoCreditLimit">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>280</y>
              <width>200</width>
              <height>40</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <fontweight>ExtraBold</fontweight>
             </font>
            </property>
            <property name="text">
             <string>0 €</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="labelSaldoLuottoText">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>350</y>
              <width>200</width>
              <height>40</height>
             </rect>
            </property>
            <property name="text">
             <string>Luottoa jäljellä:</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelSaldoLuottoaJaljella">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>350</y>
              <width>200</width>
              <height>40</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <fontweight>ExtraBold</fontweight>
             </font>
            </property>
            <property name="text">
             <string>0 €</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="screenNostaValitse">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>60</y>
              <width>200</width>
              <height>90</height>
             </rect>
            </property>
            <property name="text">
             <string>Nosta summa</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnNosta20">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>180</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="text">
             <string>20 €</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnNosta50">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>180</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="text">
             <string>50 €</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnNosta40">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>260</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="text">
             <string>40 €</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnNosta100">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>260</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="text">
             <string>100 €</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="labelNostosumma">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>340</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="btnNostaMuu">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>340</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="text">
             <string>Muu</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>390</y>
              <width>400</width>
              <height>40</height>
             </rect>
            </property>
            <property name="text">
             <string>Huom. Automaatissa vain 20€ ja 50€ seteleitä</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelNostaValitseVirhe">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>450</y>
              <width>290</width>
              <height>30</height>
             </rect>
            </property>
            <property name="text">
             <string>Virheellinen summa</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelNostaValitseKate">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>420</y>
              <width>120</width>
              <height>30</height>
             </rect>
            </property>
            <property name="text">
             <string>Kate ei riitä</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelNostaValitseVirhe_2">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>480</y>
              <width>180</width>
              <height>30</height>
             </rect>
            </property>
            <property name="text">
             <string>Virheellinen summa</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="screenNostaVahvista">
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>210</y>
              <width>210</width>
              <height>40</height>
             </rect>
            </property>
            <property name="text">
             <string>Olet nostamassa:</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelNostaVahvistaSumma">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>210</y>
              <width>200</width>
              <height>40</height>
             </rect>
            </property>
            <property name="text">
             <string>0 €</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="btnNostaVahvistaVahvista">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>370</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string>Vahvista</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="screenTapahtumat">
           <widget class="QPushButton" name="btnTapahtumatVasen">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>400</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnTapahtumatOikea">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>400</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
           <widget class="QFrame" name="logsCard">
            <property name="geometry">
             <rect>
              <x>5</x>
              <y>19</y>
              <width>471</width>
              <height>371</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>8</number>
             </property>
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>8</number>
             </property>
             <property name="bottomMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="QTableView" name="tableTapahtumat">
               <property name="selectionMode">
                <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="screenLogOut">
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>180</y>
              <width>420</width>
              <height>90</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Haluatko varmasti kirjautua ulos?</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnLogOutVahvista">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>370</y>
              <width>108</width>
              <height>48</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string>Vahvista</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="footerBar">
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <layout class="QHBoxLayout" name="footerLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>14</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>14</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnSaldo">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Saldo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNostaRahaa">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Nosta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTapahtumat">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Tapahtumat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLogOut">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Kirjaudu Ulos</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="bottomSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>140</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
